### ISSUE TITLE
Active Navbar Link should be visible

### ISSUE THREAD

**Issue by aryad14** [mlscgescoe/eventGuru#4]
## Issue
The links do not have a active state, hence it is difficult to know on which route a user is.

![image](https://github.com/user-attachments/assets/ebaa2065-614e-4989-89cf-086e11f0b647)

## Expected Solution
Add some styling to the navigation link which states on which route the user is. For example, if the user is on About page, the  **About** link on the navbar should have some styling denoting that the user is currently on the About Page.

Thank you!

**Comment by PALLAVIKHEDLE** [comment]
@aryad14 Can you please assign it to me? It's pretty straight forward. I can easily knock this out.

**Comment by Rahulagg13** [comment]
Hey can you assign this issue to me ?

**Comment by dexterousdhruv** [comment]
@aryad14 Can you please assign it to me ? I'd like to work on it.

**Comment by PALLAVIKHEDLE** [comment]
@aryad14 
<img width="1111" alt="Screenshot 2024-10-08 at 10 53 28 AM" src="https://github.com/user-attachments/assets/f06de357-7672-486f-b4a2-5f5a74b9b01b">

It's not working. Do you guys use a clerk publishable key or any specific node version?
Shall I create my .env.local?

Looks like .env.local in git ignore

**Comment by PALLAVIKHEDLE** [comment]
@aryad14 any update?

**Comment by aryad14** [comment]
@PALLAVIKHEDLE I have added .env.sample file to the repo, make sure to check it and setup clerk authentication on your side
The issue you are getting is because the project needs clerk secret keys as the entire project is wrapped around the **ClerkProvider**

**Comment by PALLAVIKHEDLE** [comment]
Yes, I set up the Clerk authentication. But what about stripe and MongoDB URI?

**Comment by aryad14** [comment]
For setting up the project you have to add the mongodb url and stripe API keys. Otherwise it won't work

**Comment by PALLAVIKHEDLE** [comment]
Can you provide me those, or do you want me to add those from my end?

**Comment by aryad14** [comment]
You can add them on your side. 
Using the .env.sample file, create your own .env file in the project you cloned. 
.env and .env.local files are never pushed on GitHub, hence they should always be included in the .gitignore file
These files are only for development environment
